### PR TITLE
Tweaks A LOT of hairstyles, adds some new ones

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Please refer to the icon diff for a complete list. Mostly increases/decreases volume or tones down the shading. The recently ported baystation hairstyles have received alternative variants. 
Adds 5 new hairstyles: Neat; Neat Long; Shoulder Bob; Messy; Drills, Side. 
Removes the Father hairstyle. Please. We need to remove it. Please.

### CODE REVIEW

**Comment by VTCobaltblood** [comment]
!wip

**Comment by VTCobaltblood** [comment]
Also essentially resprites and renames Bedhead 3 into Neat, and adds Neat Long.

**Comment by VTCobaltblood** [comment]
!wip

**Comment by VTCobaltblood** [comment]
!unwip

**Comment by VTCobaltblood** [comment]
!review

**Comment by VTCobaltblood** [comment]
Also adds Shoulder Bob, requested by a player.

**Comment by VTCobaltblood** [comment]
!wip

**Comment by VTCobaltblood** [comment]
!review

**Comment by VTCobaltblood** [comment]
!review

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -248,10 +244,10 @@
 		icon_state = "hair_combover"
 		gender = MALE
 
-	father
-		name = "Father"
-		icon_state = "hair_father"
-		gender = MALE
+//	father				// NO. Just NO. Why did we ever have it.
```
Comment (Arrow768) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm]: Remove the uncommented block.

**Pull request merged**